Employee.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
1. Formatting is missing.
2. Emp - abbreviation is not good.
3. Suggestion, try to use camel case always in json.